core/new-gui/src/app/dashboard/type/workflow-metadata.interface.ts

```diff
@@ -1,5 +1,6 @@
 export interface WorkflowMetadata {
   name: string;
+  description: string | undefined;
```

Comment:
I wonder if this can be an optional field (`description?: string;`) so that you don't have to modify all those spec files. @zuozhiw?

Reply:
I think either `string | undefined` and `description?: string;` is fine

Reply:
The difference is using `description?` can preserve existing new instance creation calls without imposing the requirement to have `description` property for every new `Workflow` instance (see those 1-line changes of spec.ts files in this PR) but `string | undefined` does not have this effect.